packages/flutter/test/material/stepper_test.dart

```diff
@@ -527,22 +527,19 @@ void main() {
     final List<String> lines =  errorDetails.toString().split('\n');
     // The lines in the middle of the error message contain the stack trace
     // which will change depending on where the test is run.
-    expect(lines.length, greaterThan(9));
+    expect(lines.length, greaterThan(7));
     expect(
-      lines.take(9).join('\n'),
+      lines.take(7).join('\n'),
       equalsIgnoringHashCodes(
         '══╡ EXCEPTION CAUGHT BY WIDGETS LIBRARY ╞════════════════════════\n'
         'The following assertion was thrown building Stepper(dirty,\n'
         'dependencies: [_LocalizationsScope-[GlobalKey#00000]], state:\n'
         '_StepperState#00000):\n'
         'Steppers must not be nested. The material specification advises\n'
         'that one should avoid embedding steppers within steppers.\n'
-        'https://material.io/archive/guidelines/components/steppers.html#steppers-usage\n'
-        '\n'
-        'When the exception was thrown, this was the stack:'
+        'https://material.io/archive/guidelines/components/steppers.html#steppers-usage'
```

Comment:
why do we no longer report the stack?

Reply:
we do but the error message was truncated earlier as it would now also include the location causing the error.

Reply:
I truncated earlier so that I don't need to deal with conditional check if --track-widget-creation present or not, and it seems only the http link check is matter to this test. It seems overkill to test beyond that. Besides, we have plenty of tests that test the error message of stack trace.

Reply:
k